index.html

```diff
@@ -8555,113 +8624,164 @@ <h3>Processing actions</h3>
-  to <var>pressure</var>.
-
- <li><p>Let <var>tangentialPressure</var> be the result
-  of getting the property <code>tangentialPressure</code>
-  from <var>action item</var>.
-
- <li><p>If <var>tangentialPressure</var> is not <a>undefined</a>
-  and <var>tangentialPressure</var> is not a <a>Number</a> greater
-  than or equal to -1 and less than or equal to 1 return <a>error</a>
-  with <a>error code</a> <a>invalid argument</a>.
-
- <li><p>Set the <code>tangentialPressure</code> property of <var>action</var>
-  to <var>tangentialPressure</var>.
-
- <li><p>Let <var>tiltX</var> be the result
-  of getting the property <code>tiltX</code>
-  from <var>action item</var>.
-
- <li><p>If <var>tiltX</var> is not <a>undefined</a>
-  and <var>tiltX</var> is not an <a>Integer</a> greater than or
-  equal to -90 and less than or equal to 90 return <a>error</a> with
-  <a>error code</a> <a>invalid argument</a>.
-
- <li><p>Set the <code>tiltX</code> property of <var>action</var>
-  to <var>tiltX</var>.
-
- <li><p>Let <var>tiltY</var> be the result
-  of getting the property <code>tiltY</code>
-  from <var>action item</var>.
-
- <li><p>If <var>tiltY</var> is not <a>undefined</a>
-  and <var>tiltY</var> is not an <a>Integer</a> greater than or
-  equal to -90 and less than or equal to 90 return <a>error</a> with
-  <a>error code</a> <a>invalid argument</a>.
-
- <li><p>Set the <code>tiltY</code> property of <var>action</var>
-  to <var>tiltY</var>.
-
- <li><p>Let <var>twist</var> be the result
-  of getting the property <code>twist</code>
-  from <var>action item</var>.
-
- <li><p>If <var>twist</var> is not <a>undefined</a>
-  and <var>twist</var> is not an <a>Integer</a> greater than or
-  equal to 0 and less than or equal to 359 return <a>error</a> with
-  <a>error code</a> <a>invalid argument</a>.
-
- <li><p>Set the <code>twist</code> property of <var>action</var>
-  to <var>twist</var>.
-
- <li><p>Let <var>altitudeAngle</var> be the result
-  of getting the property <code>altitudeAngle</code>
-  from <var>action item</var>.
-
- <li><p>If <var>altitudeAngle</var> is not <a>undefined</a>
-  and <var>altitudeAngle</var> is not a <a>Number</a> greater than
-  or equal to 0 and less than or equal to π/2 return <a>error</a> with
-  <a>error code</a> <a>invalid argument</a>.
-
- <li><p>Set the <code>altitudeAngle</code> property of <var>action</var>
-  to <var>altitudeAngle</var>.
-
- <li><p>Let <var>azimuthAngle</var> be the result
-  of getting the property <code>azimuthAngle</code>
-  from <var>action item</var>.
+ <li><p>Let <var>pointerType</var> be equal to
+  <var>action</var>’s <code>pointerType</code> property.
+  
+ <li><p>Run the substeps of the first matching value of <var>pointerType</var>
 
- <li><p>If <var>azimuthAngle</var> is not <a>undefined</a>
-  and <var>azimuthAngle</var> is not a <a>Number</a> greater than
-  or equal to 0 and less than or equal to 2π return <a>error</a> with
-  <a>error code</a> <a>invalid argument</a>.
+  <dl class=switch>
```

Comment:
This switch needs to be exhaustive i.e. you need a `"<code>mouse</code>"` option, even if it's a no-op.